aips/aip-116.md

```diff
@@ -120,22 +112,22 @@ The `domain` field may be omitted from the `AptosSignInInput` to prevent the cre
-| plainText         | string             | ✓         | The constructed message using the standardized message format.                                                                                                                                                                                        |
-| signingMessage    | bytes              | ✓         | The signing message that conforms to the domain separated signing algorithm.                                                                                                                                                                          |
-| signature         | bytes              | ✓         | The signature of the signing message.                                                                                                                                                                                                                 |
-| signatureType     | string             | ✓         | The key scheme used to sign the message (e.g. 'ed25519', 'single_key', 'multi_key', 'multi_ed25519', etc.).                                                                                                                                           |
+| Name              | Type   | Description                                                                                                        |
+| ----------------- | ------ | ------------------------------------------------------------------------------------------------------------------ |
+| account.address   | string | The address of the user that is signing in. This field MUST match the `address` field used to construct `message`. |
+| account.publicKey | bytes  | The public key of the user that is signing in.                                                                     |
+| message           | string | The constructed SIWA message using the standardized message format.                                                |
+| signature         | bytes  | The signature of the signing message.                                                                              |
+| signatureType     | string | The digital signature scheme used to sign (e.g. 'ed25519', 'single_key', 'multi_key', 'multi_ed25519', etc.).      |
 
 If there are library representations of the `account.publicKey` and `signature`, they may be used instead of the `bytes` type (e.g. `PublicKey` and `Signature` from the `@aptos-labs/ts-sdk` package).
 
 ### SIWA Message Construction
 
 The proposed plain text representation will be an adaptation of the standards specified from [EIP-4361](https://eips.ethereum.org/EIPS/eip-4361) and [CAIP-122](https://chainagnostic.org/CAIPs/caip-122). If certains fields are not present in the `AptosSignInInput`, omit them from the SIWA message.
 
+If the following fields are not present in the `AptosSignInInput`, they must be bound by the wallet: `address`, `chainId`, `domain`, `uri`, and `version`.
```

Comment:
If the user has multiple addresses in his account, should this AIP also cover how the user may be able to switch his address to another one after seeing the sign-in prompt and noticing its the wrong address?

Or is that left as an implementation detail for wallet devs reading this?